go.sum

```diff
@@ -1,23 +1,43 @@
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
 github.com/ONSdigital/dp-api-clients-go v0.0.0-20190920133223-0b75bbb235dd h1:GKyefMVR8NpBNYkdo4AS7K6zRDIw1MQ/hEVRH3s1CEc=
 github.com/ONSdigital/dp-api-clients-go v0.0.0-20190920133223-0b75bbb235dd/go.mod h1:vx3olkUJib+ZtLL6njP8iX1fjePokXyIcitFWKQ9F/A=
+github.com/ONSdigital/dp-authorisation v0.0.0-20191018110224-d4543527d6cb h1:k58Fs56RCIHAL3GToXYwO3IfKKyIN2JvvkQHANpIZ2Q=
```

Comment:
can you run `go mod tidy` please?

Reply:
go.mod and go.sum has been tidied in the latest commit